core/controllers/reader_test.py

```diff
@@ -364,15 +364,15 @@ def setUp(self):
 
     def test_questions_are_returned_successfully(self):
         # Call the handler.
-        url = '%s?question_count=%s&skill_ids=%s' % (
-            feconf.QUESTIONS_URL_PREFIX, '1', self.skill_id)
+        url = '%s?question_count=%s&skill_ids=%s&fetch_by_mastery=%s' % (
```

Comment:
Add a test for an invalid fetch_by_mastery value

Reply:
It's tested below (line 410)